webapp-client/src/component/Category.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import '../stylesheets/Category.css';
+
+export default class Category extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      categoryName: props.categoryName,
```

Comment:
It would be better to read the name from the props only instead of storing an extra copy of the name here.

Reply:
we removed it here:) but Category is actually just a place holding component. ItemContainer will be the component replacing this and Anthony is working on it:)))